library/ix-dev/community/overseerr/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: common
+  repository: file://../../../common
+  version: 1.0.6
+digest: sha256:2f1f31c15fb7f92db141a66adbb8d23a8598727730050a3883a211763a4e5472
```

Comment:
Should I remove/regenerate this for the PR?

Reply:
Yes, this needs to be regenerated

Within the overseerr directory run
`helm dep up`